front/app/components/admin/InternalComments/index.tsx

```diff
@@ -106,22 +97,28 @@ const InternalCommentsSection = ({ postId, postType, className }: Props) => {
     setPosting(isPosting);
   };
 
-  const commentCount = post.data.attributes.comments_count;
+  const commentCount = post.data.attributes.internal_comments_count;
 
   return (
     <Box className={className || ''}>
-      <Header>
-        <Title color="tenantText" variant="h2" id="comments-main-title">
-          <FormattedMessage {...commentsMessages.invisibleTitleComments} />
-          {commentCount > 0 && <CommentCount>({commentCount})</CommentCount>}
-        </Title>
-        <StyledCommentSorting
-          onChange={handleSortOrderChange}
-          selectedCommentSort={sortOrder}
-        />
-      </Header>
+      {!!commentCount && (
```

Comment:
- This works but how about we check if commentCount is greater than 0? That describes our intention more clearly. `!!` is cryptic and a (questionable) shortcut to check if something is defined rather than to check if the number is greater than 0.
- I think it's odd to not have the Comments heading when there are no comments yet. I found its position not good before, and this gives me more reason to think so. It should always be there and above the tabs imo. But I'll discuss/take it as a separate task. Comment sorting should maybe also come right above the comments (under the parent comment form, not above it).

Reply:
The position could be revised in the back office indeed. I think showing the word comments when there are no comments is odd. But if we change the position then it might not be a problem.

On the position of sorting, I think it might be better to have that in that tabs actually. Otherwise, it is not clear what is being sorted. But we can discuss this separately in another ticket if needed.